sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -620,7 +650,7 @@ export class KeyClient {
 
       return this.getKeyFromKeyBundle(response);
     } else {
-      const response = await this.client.updateKey(this.vaultUrl, name, keyVersion, options);
+      const response = await this.client.updateKey(this.vaultUrl, name, keyVersion, {});
```

Comment:
Update key doesn't accept an empty options bag, so here we are.

Just as in the other instances, I removed "options" since by this point options is undefined. See the if conditional above in the code (you should be able to see the } else { part from here).